packages/ubuntu_desktop_installer/lib/pages/installation_complete/installation_complete.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+
+import '../../widgets.dart';
+import '../wizard_page.dart';
+import 'installation_complete_model.dart';
+
+const _kHighlightBackground = Color(0xFF0e8420);
+const _kHighlightForeground = Colors.white;
+
+class InstallationCompletePage extends StatelessWidget {
+  const InstallationCompletePage({Key? key}) : super(key: key);
+
+  static Widget create(BuildContext context) => InstallationCompletePage();
+
+  @override
+  Widget build(BuildContext context) {
+    final TextStyle? boldStyle = Theme.of(context)
+        .textTheme
+        .bodyText1!
+        .copyWith(fontWeight: FontWeight.bold);
+    return LocalizedView(
+      builder: (context, lang) => WizardPage(
+        title: Text(lang.installationCompleteTitle),
+        content: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(top: 64, bottom: 32),
+              child: Image.asset(
+                'assets/version.png',
+                width: 160,
+                height: 160,
+              ),
+            ),
+            RichText(
+              text: TextSpan(
+                  style: Theme.of(context).textTheme.bodyText1,
+                  children: TextUtils.parseText(lang.readyToUse('Ubuntu 20.04'))
```

Comment:
This should be loaded from external file, but I'm not sure where, again, open for discussion

Reply:
For ISOs, there is a file on disk (`/cdrom/.disk/info`) that contains the version and codename of the product being installed, so you could probably extract the required information from there. Something similar is done for the release notes URL, see the `_releaseNotesURL()` function in `lib/pages/try_or_install_page.dart`.

The contents of that file for a 20.04.2 release ISO downloaded from ubuntu.com is:

    Ubuntu 20.04.2.0 LTS "Focal Fossa" - Release amd64 (20210209.1)

So some careful parsing will be required to extract the relevant string.